### PR TITLE
Update Patternfly and Axios deps

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #4429 
link #4238

Updates patternfly and axios deps to latest.  I also added the `start` toggle to the notifications list while I was in there looking at something else.

You may notice this error being thrown to the console:

Screen Shot 2019-08-27 at 4 47 21 PM

In this case, it's referencing the fact that we pass the `Link` component through to patternfly's `Button` component to tell it to wrap the button in a react-router `Link`.  The code is functioning as expected.  After talking with @keithjgrant and looking at the source code I believe that this is a bug in the patternfly-react repo so I'll open up an issue there.

I'm pretty sure _all_ the snapshots had to be updated since it had been quite a while since our last upgrade and classes/components have changed a bit.

##### ISSUE TYPE
 - Task

##### COMPONENT NAME
 - UI

##### AWX VERSION
awx: 6.1.0

### CODE REVIEW

**Comment by marshmalien** [comment]
Within Pagination.jsx, we should give the `titles` prop a `page` value to resolve this screenshot below. 
Screen Shot 2019-08-28 at 11 22 14 AM
PF Pagination Code: https://github.com/patternfly/patternfly-react/blob/f494b4c13d5ed64abe0e379ee61d393a82b33740/packages/patternfly-4/react-core/src/components/Pagination/Pagination.js#L64

File: awx/ui_next/src/components/RoutedTabs/RoutedTabs.jsx [hunk 1]
```diff
@@ -30,6 +30,12 @@ const Tabs = styled(PFTabs)`
     border-width: var(--pf-c-tabs__item--BorderWidth) 0
       var(--pf-c-tabs__item--BorderWidth) 0;
   }
+
+  // The above pseudo element is being applied to the tab items as well
+  // I couldn't figure out how to only apply it to .pf-c-tabs
+  .pf-c-tabs__item::before {
+    content: none;
+  }
```
Comment (marshmalien) [inline awx/ui_next/src/components/RoutedTabs/RoutedTabs.jsx]: Something very _strange_ is happening here. The generated styled-component className is getting forwarded from the `Tabs` component to the individual `Tab` component. This solution works and I can't think of a better work-around at the moment. 👍

**Review by marshmalien — APPROVED** [review]
Awesome. Thanks for getting the UI back up-to-date with PF 💪

File: awx/ui_next/src/components/RoutedTabs/RoutedTabs.jsx (hunk 1 above)
Comment (keithjgrant) [reply awx/ui_next/src/components/RoutedTabs/RoutedTabs.jsx]: You could try changing the first selector to `&:not(.pf-c-tabs__item)::before`

**Comment by jakemcdermott** [comment]
regate

**Comment by jakemcdermott** [comment]
regate

**Pull request merged**